alexa.py

```diff
@@ -44,8 +44,13 @@
   app.config['ASK_APPLICATION_ID'] = os.getenv('MEDIA_CENTER_SKILL_ID')
 
 LANGUAGE = config.get('global', 'language')
```

Comment:
Not sure what this get is. If you can provide a default value there, do it.
Otherwise, do something like:
```
if LANGUAGE == 'None':
  LANGUAGE == 'en'
```
Which in turns simplify the if/then/else under it.

Reply:
The get retrieve the language value from kodi.config. No idea if it's possible to provide a default value, so I implemented your suggestion.

Reply:
I'll look into this. We should be able to default it to english.